docs/var.tfvars-doc.md

```diff
@@ -70,6 +70,8 @@ worker      = {memory      = "32",   processors  = "0.5",  "count"   = 2}
 
 To enable high availability (HA) for cluster services running on the bastion set the `bastion.count` value to 2. Note that in case of HA, the automation will not setup additional storage (eg: NFS). Value 1 for `bastion.count` implies the default non-HA bastion setup.
 
+The value for master.count should be greater than or equal to 3 and always an odd number. You can optionally set worker.count value to 0 in which case all the clusters pods will be running on the master nodes (need more resources).
+
```

Comment:
Change this to something like:
"The value for master.count should be 3. You can optionally set worker.count value to 0 in which case all the clusters pods will be running on the master/supervisor nodes. Ensure you use proper sizing for master/supervisor nodes to avoid resource starvation for containers"

We haven't tested with 5 masters. So unless it's tested, I just want to stick to 3

Reply:
The number of masters support should come along with OCP, right? If we have not tested then I suggest I remove that statement altogether. What say?